  0%|                                                                                                                     | 0/1000000 [00:00<?, ?it/s]
TypeError: only size-1 arrays can be converted to Python scalars
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "gym_train.py", line 243, in <module>
    args.no_tqdm
  File "gym_train.py", line 93, in train
    replay_buffer.add(state, action, next_state, reward, done)
  File "/Users/kiranganeshan/reading-group/RL/util.py", line 209, in add
    self.action[self.ptr] = action
ValueError: setting an array element with a sequence.